### PR TITLE
[TOOLS-1506] WIP KPI Chart

### PR DESCRIPTION
**Pull request by heronoyama**
- Refactoring to transform IssueStatusFlowFactoy into Service
- WIP KPI Chart Back-end
- WIP KPI Chart Front-emd

### CODE REVIEW

File: src/test/java/objective/taskboard/followup/ThroughputKPIServiceTest.java [hunk 1]
```diff
@@ -48,6 +56,39 @@ public void setupProperties() {
         
         when(jiraProperties.getFinalStatuses()).thenReturn(finalStatuses);
         
+        IssueStatusFlow demand = new IssueStatusFlowBuilder("I-1")
```
Comment (taksan) [inline src/test/java/objective/taskboard/followup/ThroughputKPIServiceTest.java]: Why is a Throughput test changed in a WIP PR?

File: src/test/java/objective/taskboard/followup/WipKpiServiceTest.java [hunk 2]
```diff
@@ -45,9 +53,44 @@ public void setupProperties() {
         statusCountingOnWip.setSubtasks(new String[] { "Doing"});
 
         when(jiraProperties.getStatusCountingOnWip()).thenReturn(statusCountingOnWip);
+        
+        IssueStatusFlow demand = new IssueStatusFlowBuilder("I-1")
```
Comment (taksan) [inline src/test/java/objective/taskboard/followup/WipKpiServiceTest.java]: Why the WipKPIServiceTest changed if the PR was supposed only to expose data to the chart?

File: src/test/java/objective/taskboard/followup/ThroughputKPIServiceTest.java (hunk 1 above)
Comment (heronoyama) [reply src/test/java/objective/taskboard/followup/ThroughputKPIServiceTest.java]: Refactoring i made in the IssueStatusFactory

File: src/test/java/objective/taskboard/followup/WipKpiServiceTest.java (hunk 2 above)
Comment (heronoyama) [reply src/test/java/objective/taskboard/followup/WipKpiServiceTest.java]: Another change related to the refactoring

**Pull request merged**